bin/publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+#
+# set version and publish to registry
+#
+
+# pipelines env
+branch=${BITBUCKET_BRANCH?}
+repoSlug=${BITBUCKET_REPO_SLUG?}
+
+# read current version
+version=$(npm show $repoSlug version)
+major="$(echo $version | cut -d'.' -f1)"
+minor="$(echo $version | cut -d'.' -f2)"
+patch="$(echo $version | cut -d'.' -f3 | cut -d'-' -f1)"
+
+# increment patch version
+patch="$(($patch + 1))"
+newVersion="$major.$minor.$patch"
+
+# set prerelease suffix for non-master builds
+if [ $branch != "master" ]; then
```

Comment:
Good point. I've added this to the PR now. The code has been tested and seems to work.